### DISCUSSION TITLE
Continuous 406 Errors

### DISCUSSION THREAD

**Discussion by jonathan-politzki** [supabase/supabase#30386] [Questions]
I have 406 error over and over again. I don't know what to do to get my supabase datbase interacting with Oauth with LinkedIn and Google. It seems like a simple error but I am completely incapacitated by it.

**Comment by ssudaraka** [comment]
Hi @jonathan-politzki,

What's the error you're seeing? Can you please provide a bit more context on your application and how you're initiating auth? Do you see any errors in your auth logs (https://supabase.com/dashboard/project/_/logs/auth-logs) when authenticating with LinkedIn?
